openedx_proversity_reports/edxapp_wrapper/backends/student_i_v1.py

```diff
@@ -1,7 +1,16 @@
 """ Backend abstraction """
+from openedx.core.djangoapps.user_api.accounts.serializers import \
```

Comment:
Let's use something like:
```suggestion
from openedx.core.djangoapps.user_api.accounts.serializers import (
    UserReadOnlySerializer,
)
```

Reply:
Addressed. @diegomillan